src/Umbraco.Web.Common/ImageProcessors/CropWebProcessor.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Microsoft.Extensions.Logging;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Processing;
+using SixLabors.ImageSharp.Web;
+using SixLabors.ImageSharp.Web.Commands;
+using SixLabors.ImageSharp.Web.Processors;
+using ImageCropperCropCoordinates = Umbraco.Cms.Core.PropertyEditors.ValueConverters.ImageCropperValue.ImageCropperCropCoordinates;
+
+namespace Umbraco.Cms.Web.Common.ImageProcessors
+{
+    public class CropWebProcessor : IImageWebProcessor
+    {
+        /// <summary>
+        /// The command constant for the crop definition
+        /// </summary>
+        public const string Crop = "crop";
+
+        public FormattedImage Process(
+            FormattedImage image,
+            ILogger logger,
+            IDictionary<string, string> commands,
+            CommandParser parser,
+            CultureInfo culture)
+        {
+            ImageCropperCropCoordinates cropCoordinates = GetCropCoordinates(commands, parser, culture);
```

Comment:
Use the most common result.
```c#
ImageCropperCropCoordinates cropCoordinates = GetCropCoordinates(commands, parser, culture);

if (cropCoordinates != null)
{
    Size size = image.Image.Size();
    Rectangle crop = GetCropRectangle(size.Width, size.Height, cropCoordinates);
    image.Image.Mutate(x => x.Crop(crop));
}

return image;
```

Reply:
Agree, having a single return statement at the bottom of the function is a lot more readable 😃